.changeset/fifty-owls-check.md

```diff
@@ -0,0 +1,5 @@
+---
+'xstate': patch
+---
+
+Provide convenience type InterpreterFrom<typeof machine>
```

Comment:
```suggestion
Provide a convenience type for getting the `Interpreter` type based on the `StateMachine` type by transferring all generic parameters onto it. It can be used like this: `InterpreterFrom<typeof machine>`
```